### DISCUSSION TITLE
I have been able to get docker and fast api set up and running, what are the next steps?

### DISCUSSION THREAD

**Discussion by phynicz** [fastapi/full-stack-fastapi-template#1819] [Questions]
The readme files albeit informative leaves some details out for newbies, I want to know how to write a backend and a frontend using this API, do I have to replace any files or just update the already existing files,for  example whenever I load localhost/api it shows details not found, how do I change this to show my actual backed and the front end how do I get it to show what I want.

**Comment by phynicz** [comment]
what do i do with the postgres server do i create a database, there are so many abstractions.

**Comment by pramadito** [comment]
there 2 things you can do next:

1. Deploy this in real production [check out this medium article](https://tiangolo.medium.com/docker-swarm-mode-and-traefik-for-a-https-cluster-20328dba6232).  or trying to deploy the smaller one for free [check this one](https://towardsdatascience.com/how-to-deploy-your-fastapi-app-on-heroku-for-free-8d4271a4ab9) and  [this one](https://levelup.gitconnected.com/building-a-website-starter-with-fastapi-92d077092864) here's example of mine [https://simple-fastapi-template.herokuapp.com/](simple-fastapi-template.herokuapp.com)
2. Learn How Vue works for frontend and postgres server works

also what goals do you want to make this app?

**Comment by phynicz** [comment]
Thank you for your response, I will learn Vue and Postgres server.
Although I want to learn fast API for skill purposes, I also want to currently use it to monitor data and changes in the data to sort of create an integrated data management/ monitoring system. Probably integrating another API with it as well.

**Comment by phynicz** [comment]
Also, with the vue JS, do I have to input a new file with the code or modify already existing frontend files..

**Comment by pramadito** [comment]
you can do both with Vue,js, 

just keep in mind that vue js has higher entry barrier for beginner to learn but really easy to learn after that and scale up to very big production.

**Comment by csouflis** [comment]
Just to be clear, what's the best way to go about developing this repo? Like what's a clear path to learning to improving and changing the customizable parts? I appreciate any feedback!

**Comment by ieferrari** [comment]
I think that the main point is to make the front-end interact with all the api's exposed in localhost:80/docs  by adding more views in 
full-stack-fastapi-postgresql/frontend/src/router.ts

it would be nice to have an example to copy and paste for that kind of simple interaction with Vue
